### ISSUE TITLE
city split of so2sat

### ISSUE THREAD

**Issue by shermanlian** [insitro/ChannelViT#15]
Hi,

Great work on the project! We’re currently reproducing your results and have a couple of questions about the So2Sat-LCZ42 implementation:
	1.	When you refer to the “city split,” does that mean the the first version (only training/validation split) from the original dataset (https://github.com/zhu-xlab/So2Sat-LCZ42 First version)?
	2.	Are the “city split” results in your main table computed on the validation set or on the official test set of So2Sat-LCZ42?

Thanks for your help!
